### PR TITLE
stability: add sst-file-corruption case

### PR DESCRIPTION
**Pull request by zyguan**
This PR test tikv failover by attaching to a tikv store and truncating a sst file. The failover should be triggered in a short time (5min) after `maxStoreDownTime+tikvFailoverPeriod`.
see: https://github.com/pingcap/tidb-operator/pull/368 & https://github.com/pingcap/tidb-operator/issues/362

### CODE REVIEW

File: tests/pkg/util/client/client.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
+// Copyright 2019 PingCAP, Inc.
```
Comment (weekface) [inline tests/pkg/util/client/client.go]: move this file to `tests/pkg/client/client.go`?

File: tests/pkg/util/ops/tikv.go [hunk 2]
```diff
@@ -0,0 +1,102 @@
+// Copyright 2019 PingCAP, Inc.
```
Comment (weekface) [inline tests/pkg/util/ops/tikv.go]: move all these file from `util` to `tests/pkg`?

File: tests/pkg/ops/common.go [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+const DefaultPoll = 10 * time.Second
+const DefaultTimeout = 10 * time.Minute
+
+type PollFn func(conditionFunc wait.ConditionFunc) error
+
+func PollImmediate(ds ...time.Duration) PollFn {
```
Comment (xiaojingchen) [inline tests/pkg/ops/common.go]: what is this function mean?

**Comment by weekface** [comment]
@xiaojingchen ptal

**Pull request merged**